ambariclient/models.py

```diff
@@ -238,13 +238,16 @@ def _server_components(self):
 
         return server_components
 
-    def install(self):
+    def install(self, context):
```

Comment:
You should make these not required by either doing:
```context=None```
or just putting the default string in the parameter list:
```context="Install All Host Components"```